exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0){
+    return `${number} is negative`
```

Comment:
The question ask to return negative or positive, I think you don't have to return a sentence here